pyogrio/tests/test_arrow.py

```diff
@@ -582,7 +588,7 @@ def test_write_append_unsupported(tmpdir, naturalearth_lowres, driver, ext):
 def test_write_gdalclose_error(naturalearth_lowres):
     meta, table = read_arrow(naturalearth_lowres)
 
-    filename = "s3://non-existing-bucket/test.geojson"
+    filename = "/vsis3/non-existing-bucket/test.geojson"
```

Comment:
What was the reason for this change? (we should at least also ensure writing to s3:// works as well, so maybe parametrize the test for both options?)

Reply:
These were producing failures when I had disabled handling the vsi paths; since that has been reverted I'm also reverting these.